src/components/DataTable/DataTable.tsx

```diff
@@ -428,14 +439,14 @@ export const DataTable = (props: IDataTableProps) => {
 													!_.isNil(columnProps.hasBorderRight)
 														? columnProps.hasBorderRight
 														: isFixedColumn &&
-														columnIndex + 1 + (isSelectable ? 1 : 0) ===
-														endColumn
+														  columnIndex + 1 + (isSelectable ? 1 : 0) ===
+																endColumn
```

Comment:
i'm worried about this indentation. i assume it was auto-indented but still looks a little weird

Reply:
Yes, it's the linter... But should be fine